manifests/init.pp

```diff
@@ -37,6 +42,14 @@
   $mesh_ipv6_prefixlen = ip_prefixlen($mesh_ipv6)
   $mesh_ipv6_address   = ip_address($mesh_ipv6)
 
+  # set default interface names
+  if $mesh_interface == "" {
+    $mesh_interface = "${mesh_code}-mesh-vpn"
```

Comment:
simply choosing a new naming scheme breaks monitoring on our site (monitoring system has to be adapted, graphs start empty again, ...). with my change it is possible to set an interface name while keeping the known default.